### PR TITLE
Update modern-root-site.md

### PR DESCRIPTION
**Pull request by WayneEwington**
Include the Site Rename smart gesture functionality invoked by default when performing a Site Swap.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/modern-root-site.md [hunk 1]
```diff
@@ -49,14 +49,16 @@ If you've [turned on audit log search](/office365/securitycompliance/turn-audit-
 - Scheduled site swap: A site swap was scheduled at this time
 - Swapped site: A site swap completed successfully at this time
 - Failed site swap: A site swap failed at this time and won't be tried again
+
+By default, a site redirect will be created that will redirect traffic from the source site to the root site. [Manage site redirects](manage-site-redirects).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/modern-root-site.md]:
```suggestion
By default, a site redirect will be created that will redirect traffic from the source site to the root site. See [Manage site redirects](manage-site-redirects) for more information.
```

Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/modern-root-site.md]:
```suggestion
By default, a site redirect will be created that will redirect traffic from the source site to the root site. Please see [Manage site redirects](manage-site-redirects.md) for more information.
```
Please confirm that link is correct.

File: SharePoint/SharePointOnline/modern-root-site.md [hunk 2]
```diff
@@ -49,14 +49,16 @@ If you've [turned on audit log search](/office365/securitycompliance/turn-audit-
...
  
 ### Limitations
 
 - Only sites within the same domain, for example, https://contoso.sharepoint.com can be swapped.
 - The source site must be a modern team site (STS#3), a communication site (SITEPAGEPUBLISHING#0), or a classic team site (STS#0).
 - All subsites contained with the source and target sites will be swapped.
 - The source and target sites can't be connected to an Office 365 group. They also can't be hub sites or associated with a hub.<br>If a site is a hub site, unregister it as a hub site, swap the root site, and then register the site as a hub site. If a site is associated with a hub, disassociate the site, swap the root site, and then reassociate the site. [Learn how to manage hubs in the new SharePoint admin center](manage-sites-in-new-admin-center.md#change-a-sites-hub-site-association)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/modern-root-site.md]:
```suggestion
- The source and target sites can't be connected to an Office 365 group. They also can't be hub sites or associated with a hub.<br>If a site is a hub site, unregister it as a hub site, swap the root site, and then register the site as a hub site. If a site is associated with a hub, disassociate the site, swap the root site, and then reassociate the site. [Learn how to manage hubs in the new SharePoint admin center](manage-sites-in-new-admin-center.md#change-a-sites-hub-site-association).
```

File: SharePoint/SharePointOnline/modern-root-site.md [hunk 3]
```diff
@@ -81,7 +83,7 @@ For more info about using this cmdlet and what happens with the previous root si
 
 - The target site might return a "not found" (HTTP 404) error for a short time.
 - Content must be recrawled to update the search index for the sites that have been swapped. This might take some time depending on factors such as the amount of content in these sites. Anything dependent on the search index might return incomplete results until the swapped sites have been recrawled.
-- Anything dependent on "static" links (such as File Sync and OneNote files) will need to be manually updated.
+- If creating a site redirect was disabled, then anything dependent on "static" links (such as File Sync and OneNote files) will need to be manually updated.
 - If the source site was an organizational news site, update the URL. [Get a list of all organizational news sites](/powershell/module/sharepoint-online/get-spoorgnewssite?view=sharepoint-ps)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/modern-root-site.md]:
```suggestion
- If the source site was an organizational news site, update the URL. [Get a list of all organizational news sites](/powershell/module/sharepoint-online/get-spoorgnewssite?view=sharepoint-ps).
```

**Comment by scanum** [comment]
@WayneEwington @kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**